webapp/src/tutorial.tsx

```diff
@@ -715,7 +715,7 @@ export class TutorialCard extends data.Component<TutorialCardProps, TutorialCard
         const isRtl = pxt.Util.isUserLanguageRtl();
         return <div id="tutorialcard" className={`ui ${tutorialStepExpanded ? 'tutorialExpanded' : ''} ${tutorialReady ? 'tutorialReady' : ''} ${this.state.showSeeMore ? 'seemore' : ''}  ${!this.state.showHint ? 'showTooltip' : ''} ${hasHint ? 'hasHint' : ''}`} style={tutorialStepExpanded ? this.getExpandedCardStyle('height') : null} >
             {hasHint && this.state.showHint && !showDialog && <div className="mask" role="region" onClick={this.closeHint}></div>}
-
+            {validationEnabled && showTutorialValidationMessage && <div className="mask" role="region" onClick={this.closeTutorialValidationMessage}></div>}
```

Comment:
```suggestion
            {validationEnabled && showTutorialValidationMessage && <div className="mask" role="region" onClick={this.closeTutorialValidationMessage} />}
```

just personal preference, completely fine to ignore; any tag can be self closing in [jt]sx so it's sometimes easier to follow when that is used~